pkg/controllers/provisioning/scheduling/topology.go

```diff
@@ -394,13 +394,29 @@ func TopologyListOptions(namespace string, labelSelector *metav1.LabelSelector)
 		selector = selector.Add(*requirement)
 	}
 	for _, expression := range labelSelector.MatchExpressions {
-		requirement, err := labels.NewRequirement(expression.Key, selection.Operator(expression.Operator), expression.Values)
+		requirement, err := labels.NewRequirement(expression.Key, mapOperator(expression.Operator), expression.Values)
 		runtime.Must(err)
 		selector = selector.Add(*requirement)
 	}
 	return &client.ListOptions{Namespace: namespace, LabelSelector: selector}
 }
 
+func mapOperator(operator metav1.LabelSelectorOperator) selection.Operator {
```

Comment:
Isn't this the default behavior of what you were doing before?

Reply:
Nope, the strings are different:

```go
package metav1
const (
	LabelSelectorOpIn           LabelSelectorOperator = "In"
	LabelSelectorOpNotIn        LabelSelectorOperator = "NotIn"
	LabelSelectorOpExists       LabelSelectorOperator = "Exists"
	LabelSelectorOpDoesNotExist LabelSelectorOperator = "DoesNotExist"
)
```
```go
package selection
const (
	DoesNotExist Operator = "!"
	Equals       Operator = "="
	DoubleEquals Operator = "=="
	In           Operator = "in"
	NotEquals    Operator = "!="
	NotIn        Operator = "notin"
	Exists       Operator = "exists"
	GreaterThan  Operator = "gt"
	LessThan     Operator = "lt"
)
```